geotrellis/geotrellis-site
# Serve documentation from docs.geotrellis.io

This PR makes the [GeoTrellis documentation](https://geotrellis.readthedocs.io) accessible from https://docs.geotrellis.io by adding an nginx container that reverse-proxies requests to https://geotrellis.readthedocs.io. I followed the method outlined at http://docs.readthedocs.io/en/latest/alternate_domains.html#cname-ssl.
# Notable changes
- Add `gtdocs-nginx` container that reverse proxies all requests to readthedocs 
- Add ECS service for `gtdocs-nginx`
- Add firewall rules to allow HTTPS ingress/egrees to the ALB and all traffic between the ALB and container instance.
- Add a Route53 record for docs.geotrellis.io

# Testing
- see https://docs.geotrellis.io

- Run `scripts/infra plan`, ensure no changes are necessary.
```
TRAVIS_COMMIT=dafb2db GT_SITE_SETTINGS_BUCKET=geotrellis-site-production-config-us-east-1 ./scripts/infra plan
```

Fixes #72
